pkg/cmd/admin/groups/new.go

```diff
@@ -49,10 +52,18 @@ func NewCmdNewGroup(name, fullName string, f *clientcmd.Factory, out io.Writer)
 				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
 			}
 
+			kcmdutil.CheckErr((kcmdutil.ValidateOutputArgs(cmd)))
+
 			kcmdutil.CheckErr(options.AddGroup())
+
+			mapper, _ := f.Object(false)
+			shortOutput := kcmdutil.GetFlagString(cmd, "output") == "name"
+			kcmdutil.PrintSuccess(mapper, shortOutput, out, "group", options.Group, "created")
```

Comment:
> kcmdutil.PrintSuccess is specifically documented as being used for mutating operations (and is used in a lot of other places in the code for this purpose).

Yeah, you just called it the wrong place, using the wrong output stream, with an unpleasant interface for the command.